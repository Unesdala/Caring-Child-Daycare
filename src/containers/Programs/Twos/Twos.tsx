import { Component } from 'react';
import commonUtils from '../../../components/lib/commonUtils';
import DefaultTwosContent from './TwosContent';

export class Twos extends Component {
  commonUtils: { setTitleAndScroll: (pageTitle: string, width: number) => void };

  constructor(props: Record<string, unknown>) {
    super(props);
    this.commonUtils = commonUtils;
  }

  componentDidMount(): void { this.commonUtils.setTitleAndScroll('Twos', window.screen.width); }

  render(): JSX.Element {
    return (
      <>
        <div className="page-content">
          <DefaultTwosContent />
        </div>
      </>
    );
  }
}

export default Twos;
