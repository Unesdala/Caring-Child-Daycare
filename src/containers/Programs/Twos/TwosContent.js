import React from 'react';

const TwosContent = () => (
  <main className="twos">
    <p />
  </main>
);

export default TwosContent;
