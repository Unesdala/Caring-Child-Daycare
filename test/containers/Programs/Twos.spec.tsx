/* eslint-disable @typescript-eslint/no-explicit-any */

import { shallow } from 'enzyme';
import { Twos } from 'src/containers/Programs/Twos/Twos';
import TwosContent from 'src/containers/Programs/Twos/TwosContent';

describe('Twos', () => {
  const wrapper = shallow(<Twos />);
  it('renders', () => {
    expect(wrapper.find(TwosContent).exists()).toBe(true);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find(TwosContent).dive().find('main.twos-main--content').exists()).toBe(true);
  });
  
});
