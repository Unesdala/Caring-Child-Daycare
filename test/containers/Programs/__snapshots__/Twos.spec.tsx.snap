// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Twos renders 1`] = `ShallowWrapper {}`;
